src/openrct2/world/Scenery.cpp

```diff
@@ -394,3 +395,41 @@ void RestrictAllMiscScenery()
         }
     }
 }
+
+ObjectType GetObjectTypeFromSceneryType(uint8_t type)
```

Comment:
These are only used by the parkfile and then only for legacy interop. Suggest moving them back.